examples/pwa-app/i18n/en.pot

```diff
@@ -0,0 +1,19 @@
+msgid ""
+msgstr ""
+"Project-Id-Version: i18next-conv\n"
+"MIME-Version: 1.0\n"
+"Content-Type: text/plain; charset=utf-8\n"
+"Content-Transfer-Encoding: 8bit\n"
+"Plural-Forms: nplurals=2; plural=(n != 1)\n"
+"POT-Creation-Date: 2021-06-08T14:28:10.339Z\n"
+"PO-Revision-Date: 2021-06-08T14:28:10.339Z\n"
+
+msgid ""
+"App updates are ready and will be activated after all tabs of this app are "
+"closed. Skip waiting and reload to update now?"
+msgstr ""
+"App updates are ready and will be activated after all tabs of this app are "
+"closed. Skip waiting and reload to update now?"
+
+msgid "Update"
+msgstr "Update"
```

Comment:
These strings are defined from `sw/src/offline-interface/OfflineInterface.js`

Reply:
Does this mean that `@dhis2/sw` depends on `@dhis2/d2-i18n` ?

I wonder if these strings are extracted because of the `file://` directive in `shell/package.json` and how this would work with standard dependency resolution. I don't think that we extract translations from `node_modules`. 
Not sure about this, something to investigate.

In other places, a library owns it's own translations, but here the app owns the library's translations. That's not ideal.

Reply:
Good question! 👍  It turns out these were actually old strings that were once in the app, but the example app no longer has any translate strings. `en.pot` has now been removed